src/Repositories/Repository.cs

```diff
@@ -23,9 +23,9 @@ public TEntity Get(int id)
             return Context.Set<TEntity>().Find(id);
         }
 
-        public IEnumerable<TEntity> GetAll()
+        public IQueryable<TEntity> GetAll()
```

Comment:
I think this might be a good change on how we currently manage object collections.
IQueryable interfaces LINQ to SQL, so it might prevent us from getting too many rows into memory prematurely